### PR TITLE
Fixed errors

### PR DESCRIPTION
**Pull request by SmritiSatyanV**
Grammar fix
Fixed directive fast registration link
Signed-off-by: SmritiSatyanV <smriti@union.ai>

### CODE REVIEW

File: rsts/concepts/admin.rst [hunk 1]
```diff
@@ -59,7 +59,7 @@ The managers utilize additional components to process requests. These additional
 Repository
 ----------
 Serialized entities (tasks, workflows, launch plans) and executions (workflow-, node- and task-) are stored as protos defined
-`here <https://github.com/flyteorg/flyteidl/tree/master/protos/flyteidl/admin>`__.
+`here <admin>`__.
```
Comment (samhita-alla) [inline rsts/concepts/admin.rst:62]: Which link is this pointing to?

Comment (SmritiSatyanV) [reply rsts/concepts/admin.rst:62]: oh its not pointing correctly now. I remember seeing flyteidl having a reference in intersphinx mapping page. Fixing it now.

File: rsts/deployment/ideal_flow.rst [hunk 2]
```diff
@@ -40,4 +40,4 @@ Thus, everything is automatically tracked. In fact, multiple users can create is
 Implementing DevOps when running Flyte pipelines could be essential when Flyte is deployed to production.
 You could have a team of engineers working on various pipelines with continuous code iteration and deployment.
 In such a case, we highly encourage you to incorporate automation to build Flyte pipelines.
-This could help speed up your development, iteration, and deployment time!
\ No newline at end of file
+This could help reduce the time consumed by development, iteration, and deployment!
```
Comment (samhita-alla) [inline rsts/deployment/ideal_flow.rst:43]: Aren't both the same?

**Review by samhita-alla — APPROVED** [review]

File: rsts/deployment/ideal_flow.rst (hunk 2 above)
Comment (SmritiSatyanV) [reply rsts/deployment/ideal_flow.rst:43]: I think this provides better readability.

**Pull request merged**